Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/InvoluntaryProject/SearchSchool.cshtml.cs

```diff
@@ -0,0 +1,55 @@
+using Dfe.PrepareConversions.Data.Services;
+using Dfe.PrepareConversions.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Dfe.PrepareConversions.Pages.InvoluntaryProject
+{
+   public class SearchSchoolModel : PageModel
+	{
+		private readonly IGetEstablishment _getEstablishment;
+		private readonly ErrorService _errorService;
+
+		public SearchSchoolModel(IGetEstablishment getEstablishment, ErrorService errorService)
+		{
+			_getEstablishment = getEstablishment;
+			_errorService = errorService;
+		}
+
+		[BindProperty(Name = "query", SupportsGet = true)]
+		public string SearchQuery { get; set; } = "";
+
+		public IActionResult OnGet(bool change = false)
+		{
+			ViewData["ChangeLink"] = change;
+
+			return Page();
+		}
+
+		public async Task<IActionResult> OnPost()
+		{
+			if (string.IsNullOrWhiteSpace(SearchQuery))
+			{
+				ModelState.AddModelError(nameof(SearchQuery), "Enter the school name");
```

Comment:
"..or school URN"?

Reply:
Good point. I'll update it.
Again this came from the story but I think its a mistake.
![image](https://user-images.githubusercontent.com/109524605/216585589-36dcb732-a055-4449-8030-5ab94dc5904a.png)

Reply:
This has been changed now.